build-script-helper.py

```diff
@@ -192,10 +191,6 @@ def invoke_swift_single_product(package_dir, swift_exec, action, product, build_
   else:
     args.extend(['--product', product])
 
-  # Tell SwiftSyntax that we are building in a build-script environment so that
-  # it does not need to rebuilt if it has already been built before.
-  env['SWIFT_BUILD_SCRIPT_ENVIRONMENT'] = '1'
```

Comment:
Are you removing this from all the build script helpers? Will it just be set in solely `swift-syntax/SwiftSyntaxDevUtils` now?

Reply:
Yes, and then I’ll probably rename it to something more descriptive. This used to be used to set `--check-exclusivity=unchecked`, which decreased compile time of swift-syntax by 2 minute at some point but that’s no longer the case.